src/components/containers/Navbar.container.tsx

```diff
@@ -0,0 +1,42 @@
+import { connect } from 'react-redux'
+import { Action } from 'redux'
+
+import { Actions } from 'actions'
+import { User, FrostState } from 'interfaces/Props'
+
+import { Navbar } from '../molecules/Navbar/Navbar'
+
+export interface NavbarContainerProps {
+  readonly shadow?: boolean
+  readonly transparent?: boolean
+  readonly margin?: boolean
+  readonly displayLogo?: boolean
+  readonly displaySearch?: boolean
+  readonly searchShadow?: boolean
+}
+
+export interface LogoutProps {
+  readonly user?: User
+  readonly onSignOut?: () => Action
+}
+
+const mapStateToProps = (
+  state: FrostState,
+  ownProps: NavbarContainerProps & LogoutProps
+): NavbarContainerProps & LogoutProps => ({
+  user: state.user,
+  ...ownProps,
+})
+
+const mapDispatchToProps = {
+  dispatchSearchChange: (query: string) => ({
```

Comment:
This should use an action creator if there is one. If not, one should be created.
We want to prevent creating raw actions in different parts of the app as it makes
refactors difficult should the shape of the action change.

Reply:
OK. This was already in there and it isn't being used yet, so would you want it created in this PR or create a ticket?

Reply:
This looks like a new file so I'm not sure how it was in there before. I'd go ahead and fix it as part of this PR.

Reply:
It looks like a new file because of separating out the Navbar component into the container/ stateless component.